### PR TITLE
Update to transition from BigInt to {ui}128

### PR DESCRIPTION
**Pull request by graydon**
Absorb changes to XDR in https://github.com/stellar/stellar-xdr/pull/57 and https://github.com/stellar/stellar-xdr/pull/58 -- removing BigInt and adding I128 and U128

### CODE REVIEW

**Review by leighmcculloch — APPROVED** [review]

File: src/scval_conversions.rs [hunk 1]
```diff
@@ -1,15 +1,10 @@
-use crate::{
-    AccountId, ScBigInt, ScMap, ScMapEntry, ScObject, ScStatic, ScStatus, ScSymbol, ScVal, ScVec,
-};
+use crate::{AccountId, ScMap, ScMapEntry, ScObject, ScStatic, ScStatus, ScSymbol, ScVal, ScVec};
 
 #[cfg(all(not(feature = "std"), feature = "alloc"))]
 extern crate alloc;
 #[cfg(all(not(feature = "std"), feature = "alloc"))]
 use alloc::{string::String, vec, vec::Vec};
```
Comment (leighmcculloch) [inline src/scval_conversions.rs:6]: Can you add conversions for u128 and i128 here similar to the 64bit ones?

Comment (leighmcculloch) [reply src/scval_conversions.rs:6]: Why: ScVal to and from conversions are needed in the SDK for all types as there are things like message signing in tests that rely on them. Without this the types won't be usable in UDTs that require all their fields to be convertible.

Comment (leighmcculloch) [reply src/scval_conversions.rs:6]: See line 113+.

Comment (graydon) [reply src/scval_conversions.rs:8]: added

Comment (leighmcculloch) [reply src/scval_conversions.rs:8]: Thanks!

Comment (sisuresh) [reply src/scval_conversions.rs:8]: Looks like clippy has issues with the casts in the last commit.

**Pull request merged**